File: WorldFeed/Server/src/WorldFeed.Common/Models/Properties/TimeZone.cs
namespace WorldFeed.Common.Models.Properties
{
    using Newtonsoft.Json;

    using global::WorldFeed.Common.JsonConverters;
    using global::WorldFeed.Common.Public.Models.Interfaces;

    public class TimeZone : ITimeZone
    {
        [JsonProperty("name")]
        public string Name { get; set; }

        [JsonProperty("tzinfo_name")]
        public string TzinfoName { get; set; }

        [JsonProperty("utc_offset")]
        [JsonConverter(typeof(JsonPropertyConverterRepository))]
        public int UtcOffset { get; set; }
    }
}

File: WorldFeed/Client/src/app/core/Core/Models/Properties/TimeZone.ts
import {ITimeZone} from "../../../Public/Models/Interfaces/ITimeZone";

export class TimeZone implements ITimeZone {
  // [JsonProperty("name")]
  public name: string;

  // [JsonProperty("tzinfo_name")]
  public tzinfoName: string;

  // [JsonProperty("utc_offset")]
  // [JsonConverter(typeof(JsonPropertyConverterRepository))]
  public utcOffset: number;
}
